.travis.yml

```diff
@@ -5,6 +5,7 @@ cache:
   directories:
     - "node_modules"
 script:
+- yarn lint
```

Comment:
I think linting should go last so we can see if other things have passed properly.